src/wallet_server.cpp

```diff
@@ -0,0 +1,125 @@
+#include "wallet_server.h"
+#include "backtrace.h"
+#include "requests.h"
+#include "requestdefines.h"
+
+#include <boost/program_options.hpp>
+#include <boost/property_tree/ini_parser.hpp>
+
+const int WALLET_DISK_CACHES_UPDATE_TIME_MS = 10 * 60 * 1000; //TODO: move to config
+
+namespace po = boost::program_options;
+
+namespace graft {
+
+WalletServer::WalletServer()
+{
+}
+
+WalletServer::~WalletServer()
+{
+}
+
+void WalletServer::registerWalletRequests(ConnectionManager& httpcm)
+{
+    Router router;
+
+    graft::registerWalletRequests(router, *m_walletManager);
+
+    httpcm.addRouter(router);
+}
+
+void WalletServer::setHttpRouters(ConnectionManager& httpcm)
+{
+    Router dapi_router("/dapi/v2.0");
+    auto http_test = [](const Router::vars_t&, const Input&, Context&, Output&)->Status
+    {
+        std::cout << "blah-blah" << std::endl;
+        return Status::Ok;
+    };
```

Comment:
It is a trash I think. We have already such.

Reply:
Fixed